ANSWERS.md

```diff
@@ -1,7 +1,15 @@
 <!-- Answers to the Short Answer Essay Questions go here -->
 
 1. In Mocha, what are the differences between `before`, `after`, `beforeEach`, and `afterEach`? When do they run? What are they used for?
+Before: Actions that happen before a block occurs
+After: Actions that happen after a block occurs
+They are used to set up testing
+
 
 2. What is the point of Test Driven Development? What do you personally think about this approach?
+It wants you to develope towards the testing criteria.

```

Comment:
Well, Test Driven Development does not have agency - it is a technique. The goal is not to develop towards testing criteria, it is to provide adequate coverage for predictable use cases. This includes the intended use as well as handling of incomplete data. The idea is to write tests first, prior to writing code. Once you have the architecture of your application in mind, then the components and functions necessary to accommodate the applications functionality can be approached. With tests, there is always the potential for false positive results. With test driven development, you write the test. With no code it fails. With code it may pass or fail the test. How will you ensure that your tests do not return false positives?